flow/plasma_vx.x.x.js

```diff
@@ -211,112 +109,98 @@ declare module '@wework-dev/plasma' {
     type?: $Values<{| CONDENSED: string, REGULAR: string |}>,
   |}> {}
 
-  // source file: src/components/Image/Image.jsx
-
-  // source file: src/components/SideNavBar/OldSideNavBarItem.jsx
-  declare export class OldSideNavBarItem extends React$Component<{|
-    children: React$Node,
-    darkBg: boolean,
-    icon: string,
-    iconSize: number,
-    iconStyle: any,
-    label: string,
-    onClick: () => mixed,
-    selected: boolean,
-  |}> {}
+  // source file: src/components/Card/Card.jsx
 
-  // source file: src/components/RadioButtonGroup/RadioButtonGroup.jsx
-  declare type Option = {|
+  // source file: src/components/Checkbox/Checkbox.jsx
+  declare export class Checkbox extends React$Component<{|
+    checked?: boolean,
+    data?: Data,
     description?: string,
     disabled?: boolean,
+    fancy?: boolean,
     icon?: string,
-    label: string,
-    value: string,
-  |};
+    id?: string,
+    indeterminate?: boolean,
+    isBold?: boolean,
+    name?: string,
+    onBlur?: () => mixed,
+    onChange?: () => mixed,
+    text: React$Node,
+    value?: boolean,
+  |}> {}
 
-  // source file: src/components/layout/FixedTop/FixedTop.jsx
+  // source file: src/components/CollapsibleCard/CollapsibleCard.jsx
+  declare export class CollapsibleCard extends React$Component<{|
+    children: React$Node,
+    collapsibleText: React$Node,
+    disabled: boolean,
+    headerText: React$Node,
+    isCollapsed: boolean,
+    styleColor: $Values<{| RED: string |}>,
+  |}> {}
 
-  // source file: src/components/TextInput/TextInput.jsx
-  declare export class TextInput extends React$Component<{|
-    value?: string,
-    type?: 'text' | 'password' | 'email' | 'url',
-    suffix?: React$Node,
-    prefix?: React$Node,
-    placeholder?: string,
-    onClick?: SyntheticMouseEventHandler<HTMLInputElement>,
-    name?: string,
-    maxLength?: number | string,
-    error?: boolean,
-    disabled?: boolean,
+  // source file: src/components/CollapsibleCard/CollapsibleCard.jsx
```

Comment:
I guess we never noticed before that some things were twice (and sometimes empty)

Reply:
you're right, let's remove them :)
69ed9d5cf0da8963bef48f1a26f84dbd241c612f